libbeat/docs/outputs/output-redis.asciidoc

```diff
@@ -0,0 +1,251 @@
+//begin inner exclude for redis
+ifndef::no-redis-output[]
```

Comment:
Can't we get rid of this `ifndef::no-redis-output[]`? If a Beat doesn't need this doc you'd just not `include` it, right?

Reply:
Yes, but I wanted to minimize the impact on other books in the library by focusing on Logstash first. Also, I do think we probably want to have tags that allow writers to include all but one or two outputs. This also helps with things like linking that might need to be conditionally rendered based on which outputs are supported. I tried to make all the conditional attributes use consistent naming conventions (at least, the ones that appear in this content), but forgot this one.